inspector/src/main.rs

```diff
@@ -0,0 +1,75 @@
+use alto_client::{consensus::Message, Client};
+use clap::{value_parser, Arg, Command};
+use commonware_cryptography::bls12381::PublicKey;
+use commonware_utils::{from_hex_formatted, SystemTimeExt};
+use futures::StreamExt;
+use std::time;
+use tracing::info;
+
+#[tokio::main]
+async fn main() {
+    // Parse arguments
+    let matches = Command::new("inspector")
```

Comment:
TODO: create a subcommand for fetching all consensus data by view and any block by digest